### PR TITLE
Update configure-upgrade-etcd.md

### PR DESCRIPTION
**Pull request by Chandu5662**
### Issue

Closes: #

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
@Chandu5662 Thanks you for your contribution.
Could you please sign the CLA before the PR can be reviewed.
You can follow the steps documented here: https://git.k8s.io/community/CLA.md

**Review by dipesh-rawat — COMMENTED** [review]
/easycla

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -444,6 +444,8 @@ either be a snapshot file from a previous backup operation, or from a remaining
    ```shell
    export ETCDCTL_API=3
    etcdctl --data-dir <data-dir-location> snapshot restore snapshot.db
+or 
+   etcdctl snapshot restore --data-dir <data-dir-location> snapshot.db
```
Comment (network-charles) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:448]: Thanks for opening this PR. 
This command is the same thing as the one in Line 446. The difference is just the arrangement.

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 2]
```diff
@@ -444,6 +444,8 @@ either be a snapshot file from a previous backup operation, or from a remaining
    ```shell
    export ETCDCTL_API=3
    etcdctl --data-dir <data-dir-location> snapshot restore snapshot.db
+or
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:447]: This line breaks the code snippet formatting, and isn't valid within a shell script.

**Comment by utkarsh-singh1** [comment]
@Chandu5662, Write the description of PR (why it is needed, or which issue it resolved), description will help reviewers to understand the changes from PR.

**Review by Arhell — COMMENTED** [review]
/easycla

**Comment by network-charles** [comment]
The contribution isn’t anything new.
I’ll suggest that you close the PR. And if you’d like to continue helping, tackle open issues having the “triage/accepted” label.

**Comment by reylejano** [comment]
Hi @Chandu5662 , I'm closing this PR for the following reasons:
- unsigned CLA which does not meet branch protection
- this change adds incorrect text in a code snippet

When you're ready to address the above two items, you can reopen this PR with `/reopen` in a comment
/close

**Pull request closed without merging**